.github/renovate.json

```diff
@@ -5,5 +5,11 @@
     ],
     "labels": [
         "dependencies"
-    ]
+    ],
+    "lockFileMaintenance": {
+        "enabled": true
+    },
+    "prCreation": "not-pending",
```

Comment:
I'm just curious how the Renovate runs the tests before it creates the pull request. Do we need to enable GitHub Actions for its branches? Because we only run tests for master and pr.

Reply:
> > I'm not too sure, but it seems to work elsewhere with just pr actions